src/main/java/seedu/address/logic/parser/CountryNoteCommandParser.java

```diff
@@ -13,29 +13,29 @@
 /**
  * Parses input arguments and creates a new CountryNoteCommand object.
  */
-public class CountryNoteCommandParser implements Parser<CountryNoteCommand> {
+public class CountryNoteCommandParser implements Parser<CountryNoteAddCommand> {
```

Comment:
Should this class name be changed to `CountryNodeAddCommandParser`?

Reply:
Yes, good find!